corehq/apps/app_manager/static/app_manager/js/summary/models.js

```diff
@@ -298,8 +298,8 @@ hqDefine('app_manager/js/summary/models',[
         self.moduleFormReference = function (formId) {
             var formData = self.formNameMap[formId];
             var template = self.readOnly
-                ? "<%= moduleName %> &rarr; <%= formName %>"
-                : "<a href='<%= moduleUrl %>'><%= moduleName %></a> &rarr; <a href='<%= formUrl %>'><%= formName %></a>"
+                ? "<%- moduleName %> &rarr; <%- formName %>"
```

Comment:
Confirming that if a module or form name had, say, a `<` in it, it'll still display the same as it did previously?

Reply:
Yep, it does. Here is an example https://staging.commcarehq.org/a/ap-test-project/apps/view/70920c6ee9136f4fc99dd5ca57f5cfe9/summary/case/

Reply:
sweet, thank you